src/procedures/the-cat-api/the-cat-api.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * The Cats Service provides access to the Cat API data from https://api.thecatapi.com/v1/.
+ *
+ * @service
+ * @category Society
+ */
+
+const { TheCatApiKey } = require("../utils/api-key");
+const { registerTypes, CAT_BREEDS } = require("./types");
+const ApiConsumer = require("../utils/api-consumer");
+registerTypes();
+
+// Cats API Url
+const catApiUrl = "https://api.thecatapi.com/v1/images/search";
+
+const TheCatApi = new ApiConsumer("TheCatApi", catApiUrl, {
+  cache: { ttl: 1 },
+});
+ApiConsumer.setRequiredApiKey(TheCatApi, TheCatApiKey);
+
+/**
+ * Get random cat image.
+ * @param {CatBreeds=} catBreed cat breed supported by API.
+ * @returns {Image} the requested image
+ */
+TheCatApi.getRandomCatImage = async function (catBreed = "") {
```

Comment:
This one's totally optional, but we could use more generic names like `getRandomImage`, `breed`, and `getBreeds` since the fact that they're cats comes either from it being in the cat service (for the rpcs) or from the input type (for `breed`). You could do the same for the dog service. But one benefit of making these alpha services is that we can workshop/refine the names later on if we want without breaking anyone's code.